src/protocol_traits/ledger.rs

```diff
@@ -29,33 +29,72 @@ pub trait LedgerView {
     fn get_hyperparams(&self) -> &HyperParams;
     fn set_hyperparams(&mut self, new: HyperParams);
 
+    /// Returns the "R" parameter from the paper, as an unsigned 32bit integer.
+    fn get_random_walks_num(&self) -> &R;
+    fn set_random_walks_num(&mut self, new: R);
+
+    /// Returns the "Tau" parameter for the phase 1 pruning.
+    fn get_tau(&self) -> &Tau;
+    fn set_tau(&mut self, new: Tau);
```

Comment:
I know this threshold has no specific name in the whitepaper, but naming it here would make the code easier to follow. Maybe something like `trust_threashold`

Reply:
The reason I haven't done it here is because I introduce the `tau` name in my end2end document, but I am happy to vocalise things here as well 😉